AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/holder/EFragmentHolder.java

```diff
@@ -261,8 +261,7 @@ public JBlock getOnDestroyViewAfterSuperBlock() {
 	}
 
 	public void clearInjectedView(IJAssignmentTarget fieldRef) {
-		JBlock block = getOnDestroyViewAfterSuperBlock();
```

Comment:
the block field will contain the generated onDestroyView on EFragments after calling super.onDestroyView(), so the assign below is for adding inject views nullification after calling super in onDestroyView().